### DISCUSSION TITLE
[Feature Request] Merged Timeline

### DISCUSSION THREAD

**Discussion by tobeon** [aeharding/voyager#661] [Ideas]
Would be really helpful if we could merge the timeline for all the signed in accounts. 

Any upvotes, comments, posts etc would automatically come from the account that's subscribed to that community.

The Liftoff! App does this quite well (as an example)

**Comment by aeharding** [comment]
This is an interesting idea - but right now, accounts are segregated by design. So I'm converting to a discussion :)
